static/icons/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/static/icons/mstile-150x150.png"/>
```

Comment:
Any time you refer to an icon in the code, you should omit the `static` directory. So this should be `/icons/mstile-150x150.png`